odevler/odev1/AdminLTE/pages/add_new_icon.php

```diff
@@ -0,0 +1,473 @@
+    <!-- sidebar: style can be found in sidebar.less -->
+    <section class="sidebar">
+      <!-- Sidebar user panel -->
+      <div class="user-panel">
+        <div class="pull-left image">
+          <img src="../dist/img/user2-160x160.jpg" class="img-circle" alt="User Image">
+        </div>
+        <div class="pull-left info">
+          <p>  <?php echo $admin_username;?></p>
+          <a href="#"><i class="fa fa-circle text-success"></i> Online</a>
+        </div>
+      </div>
+      <!-- search form -->
+
+      <!-- /.search form -->
+      <!-- sidebar menu: : style can be found in sidebar.less -->
+      <ul class="sidebar-menu" data-widget="tree">
+        <li class="header">MAIN NAVIGATION</li>
+        <li>
+          <a href="../index.php">
+            <i class="fa fa-dashboard"></i> <span>Home</span>
+      
+          </a>
+        </li>
+      
+        <li>
+          <a href="widgets.php">
+          <i class="fa fa-image"></i> <span>Images</span>
+           
+          </a>
+        </li>
+        
+      
+        <li class="">
+          <a href="forms/editors.php">
+            <i class="fa fa-edit"></i> <span>Editors</span>
+            <span class="pull-right-container">
+             
+            </span>
+          </a>
+        </li>
+        
+        
+        <li class='active'>
+          <a href="mailbox.php">
+            <i class="fa fa-file-text-o"></i> <span>Texts</span>
+      
+          </a>
+        </li>
+
+        <li>
+          <a href="examples/profile.php">
+            <i class="fa fa-user"></i> <span>Profile</span>
+      
+          </a>
+        </li>
+        
+        
+        
+      </ul>
+    </section>
+    <!-- /.sidebar -->
+  </aside>
+
+  <!-- Content Wrapper. Contains page content -->
+  <div class="content-wrapper">
+  
+
+<div class="container" id='table' style='margin-top:0px;'>
+<p style='color:red'class='icon_bilgi'>İconları güncelle ve ya değiştirirken önünde olan fa kelimesini kullanmayınız</p>
+
+<!-- The Modal -->
+
+<?php
+function dbconnect() {
+    $sql = mysqli_connect("localhost", "root", "", "admin_panel");   
+    if ($sql) {   
+        return $sql;
+    } else {   
+        return false;
+    }
+    
+}
+?>
+<form action="" method="post">
+
+<input type='text' value='' placeholder='İcon Adı' name='name'><br>
+<input type='text' value='' placeholder='İcon Kodu' name='code'><br>
+<input type='text' value='' placeholder='İcon Linki' name='link'><br>
+<input type='submit' class='submit' name='gonder' value='Güncelle'>
+
+
+
+</form>
+
+<?php
+if (isset($_POST['name'])) {
+    $conn = dbconnect();
+    $new_name = $_POST['name'];
+    $new_code = $_POST['code'];
+    $new_link = $_POST['link'];
+    htmlspecialchars($new_name, ENT_QUOTES);
+    htmlspecialchars($new_code, ENT_QUOTES);
+    htmlspecialchars($new_link, ENT_QUOTES);
+    $sql2 = "INSERT into `icons`  (`icon_name`,`icon_code`,`icon_link`) VALUES ('$new_name', '$new_code', '$new_link')";
```

Comment:
çift tırnak içinde değişken kullanırken değişkeni  {}  parantezleri içine almak daha sağlam oluyor. 

```suggestion
    $sql2 = "INSERT into `icons`  (`icon_name`,`icon_code`,`icon_link`) VALUES ('{$new_name}', '{$new_code}', '{$new_link}')";
```